changelogs/fragments/licensing.yml

```diff
@@ -0,0 +1,4 @@
+---
+bugfixes:
+  - Use the correct GPL license for plugin_utils (https://github.com/ansible-collections/community.hashi_vault/issues/276).
+  - Add missing ``simplified_bsd.txt`` file for BSD licensed content (https://github.com/ansible-collections/community.hashi_vault/issues/275).
```

Comment:
It's no longer called this :)

Reply:
ah right, I also forgot to add a changelog entry for #282 (since I don't want to put it in the ignored commit), updated in https://github.com/ansible-collections/community.hashi_vault/pull/280/commits/ce2917b23ca832c4ebe2d76e1fb95a13450e73d3